test/server/graphql/loaders/expense.test.js

```diff
@@ -0,0 +1,131 @@
+import { expect } from 'chai';
+
+import { loaders } from '../../../../server/graphql/loaders';
+import { requiredLegalDocuments, userTaxFormRequiredBeforePayment } from '../../../../server/graphql/loaders/expenses';
+import models from '../../../../server/models';
+import { LEGAL_DOCUMENT_TYPE } from '../../../../server/models/LegalDocument';
+import {
+  fakeCollective,
+  fakeExpense,
+  fakeHostWithRequiredLegalDocument,
+  fakeUser,
+} from '../../../test-helpers/fake-data';
+
+const US_TAX_FORM_THRESHOLD = 600e2;
+
+describe('server/graphql/loaders/expense', () => {
+  describe('userTaxFormRequiredBeforePayment', () => {
+    const req = {};
+
+    let host, collective;
+
+    before(async () => {
+      host = await fakeHostWithRequiredLegalDocument();
+      collective = await fakeCollective({ HostCollectiveId: host.id });
+    });
+
+    describe('requires user tax form before payment', () => {
+      it('when one expense is above threshold', async () => {
+        const loader = userTaxFormRequiredBeforePayment({ loaders: loaders(req) });
+        const expenseWithUserTaxForm = await fakeExpense({
+          amount: US_TAX_FORM_THRESHOLD + 1,
+          CollectiveId: collective.id,
+        });
+        const result = await loader.load(expenseWithUserTaxForm.id);
+        expect(result).to.be.true;
+      });
+
+      it('when the sum of multiple expenses is above threshold', async () => {
+        const user = await fakeUser();
+        const loader = userTaxFormRequiredBeforePayment({ loaders: loaders(req) });
+        const firstExpense = await fakeExpense({
+          amount: US_TAX_FORM_THRESHOLD - 100,
+          CollectiveId: collective.id,
+          FromCollectiveId: user.CollectiveId,
+          UserId: user.id,
+          type: 'INVOICE',
+        });
+        const secondExpense = await fakeExpense({
+          amount: 200,
+          CollectiveId: collective.id,
+          FromCollectiveId: user.CollectiveId,
+          UserId: user.id,
+          type: 'INVOICE',
+        });
+        const result1 = await loader.load(firstExpense.id);
+        const result2 = await loader.load(secondExpense.id);
+        expect(result1).to.be.false;
```

Comment:
I don't agree, the result should be `true` whether you're looking at the first expense or the second one. It's the sum that matters.

```suggestion
        expect(result1).to.be.true;
```